src/pages/new-claim/new-claim.tsx

```diff
@@ -0,0 +1,140 @@
+import { FormEventHandler, useState } from 'react';
+import { useParams } from 'react-router';
+import { BaseLayout } from '../../components/layout';
+import Input from '../../components/input';
+import Button from '../../components/button';
+import { isEmpty } from 'lodash';
+import { Policy } from '../../chain-data';
+import { useClaimsManager } from '../../contracts';
+import { useUserPolicyById } from '../../logic/policies';
+import { formatApi3, parseApi3 } from '../../utils';
+import globalStyles from '../../styles/global-styles.module.scss';
+import styles from './new-claim.module.scss';
+
+interface Params {
+  policyId: string;
+}
+
+interface FormState {
+  evidence: string;
+  amount: string;
+}
+
+function getValidationMessages(form: FormState, policy: Policy) {
+  const messages: ValidationMessages<FormState> = {};
+
+  if (form.evidence.trim().length === 0) {
+    messages.evidence = 'Please fill in this field';
+  }
+
+  // Don't try to parse if the user is starting to type a negative number
+  if (form.amount.replace(/-/g, '').length === 0) {
+    messages.amount = 'Please fill in this field';
+  } else {
+    try {
+      const parsed = parseApi3(form.amount);
+      if (parsed.lte(0)) {
+        messages.amount = 'This must be greater than zero';
```

Comment:
The violation message feels vague to me. WDYT?

```suggestion
        messages.amount = 'Amount must be greater than zero';
```

Reply:
Yeah sounds good